draft-kazuho-httpbis-priority.md

```diff
@@ -351,15 +353,31 @@ render textual information at an early moment.
 
 ## Reprioritization
 
-Once a client sends a request, it cannot reprioritize the corresponding response
-by using the Priority header field.  This is because an HTTP header field can
-only be sent as part of an HTTP message.
+Once a client sends a request with a priority, circumstances might change and
+mean that it is beneficial to change the priority of the response. As an
+example, a web browser might issue a prefetch request for an HTML document with
+the urgency parameter of the Priority request header field set to `background`.
+Then, when the user navigates to the HTML while prefetch is in action, it would
+send a reprioritization frame with the priority field value set to `urgency=0`.
 
-Therefore, to support reprioritization, it is necessary to define a
+However, a client cannot reprioritize a response by using the Priority header
+field.  This is because an HTTP header field can only be sent as part of an HTTP
+message. Therefore, to support reprioritization, it is necessary to define a
 HTTP-version-dependent mechanism for transmitting the priority parameters.
 
-One approach that we can use in HTTP/2 ({{?RFC7540}}) is to use a frame that
-carries the priority parameters.
+One approach that we can use in HTTP/2 ({{?RFC7540}}) and HTTP/3
```

Comment:
Instead of suggesting it being "one approach", maybe state that the frame-based design *is* the approach we have adopted?

Reply:
Agreed